R/pkg/R/utils.R

```diff
@@ -851,3 +851,12 @@ rbindRaws <- function(inputData){
   out[!rawcolumns] <- lapply(out[!rawcolumns], unlist)
   out
 }
+
+# Get basename without extension from URL
+basenameSansExtFromUrl <- function(url) {
```

Comment:
Got it - yeah it sounds good to remove compression and just copy the two lines of code.